### PR TITLE
off some ui elements based on authorization status

### PR DESCRIPTION
**Pull request by ajronn**

### CODE REVIEW

File: src/shared/components/navbar/Navbar.tsx [hunk 1]
```diff
@@ -37,6 +38,12 @@ const Navbar = () => {
           {IMPORTANT_LINKS.map(link => (
             <NavLink key={link.to} activeClassName={csx.activeLink} {...link} />
           ))}
+          <Guard.Unprotected>
```
Comment (polubis) [inline src/shared/components/navbar/Navbar.tsx:41]: This should be done via simple filter function - don't change implementation. Check for reference getLinksByAuthState in current master branch.

File: src/shared/components/navbar/sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -20,6 +22,12 @@ const Sidebar = memo(
         {IMPORTANT_LINKS.map(link => (
           <NavLink key={link.to} activeClassName={csx.activeLink} {...link} />
         ))}
+        <Guard.Unprotected>
```
Comment (polubis) [inline src/shared/components/navbar/sidebar/Sidebar.tsx:25]: This should be done via simple filter function - don't change implementation. Check for reference getLinksByAuthState in current master branch.

File: src/shared/components/navbar/utils.ts [hunk 3]
```diff
@@ -9,7 +9,5 @@ export const BASE_LINKS: Link[] = [
 ];
 
 export const IMPORTANT_LINKS: Link[] = [
-  { children: 'App', to: '/app' },
```
Comment (polubis) [inline src/shared/components/navbar/utils.ts:12]: Please revert this

File: src/shared/components/navbar/Navbar.tsx [hunk 4]
```diff
@@ -7,6 +7,7 @@ import { BASE_LINKS, IMPORTANT_LINKS } from '.';
 
 import Sidebar from './sidebar';
 import SidebarTrigger from './sidebar-trigger';
+import { Guard } from 'core/auth';
```
Comment (polubis) [inline src/shared/components/navbar/Navbar.tsx:10]: Import order and spacing

File: src/shared/components/footer/Footer.tsx [hunk 5]
```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 
+import { Guard } from 'core/auth';
```
Comment (polubis) [inline src/shared/components/footer/Footer.tsx:4]: Import order put this below ui

**Review by polubis — CHANGES_REQUESTED** [review]

File: src/shared/components/navbar/Navbar.tsx (hunk 1 above)
Comment (ajronn) [reply src/shared/components/navbar/Navbar.tsx]: done

File: src/shared/components/navbar/sidebar/Sidebar.tsx (hunk 2 above)
Comment (ajronn) [reply src/shared/components/navbar/sidebar/Sidebar.tsx]: done

File: src/shared/components/navbar/Navbar.tsx (hunk 4 above)
Comment (ajronn) [reply src/shared/components/navbar/Navbar.tsx]: done

File: src/shared/components/footer/Footer.tsx (hunk 5 above)
Comment (ajronn) [reply src/shared/components/footer/Footer.tsx:4]: done

**Review by polubis — APPROVED** [review]

**Pull request merged**